.github/workflows/docker.yml

```
@@ -18,14 +18,18 @@ defaults:
     shell: bash -xeuo pipefail {0}
 
 jobs:
-  ubuntu:
+  build:
     if: github.repository_owner == 'Homebrew'
-    name: docker (Ubuntu ${{ matrix.version }})
-    runs-on: ubuntu-latest
+    name: docker (${{ matrix.arch }} Ubuntu ${{ matrix.version }})
+    runs-on: ${{ matrix.arch == 'arm64' && 'ubuntu-24.04-arm' || 'ubuntu-latest' }}
     strategy:
       fail-fast: false
       matrix:
         version: ["18.04", "20.04", "22.04", "24.04"]
+        arch: ["x86_64", "arm64"]
```

```suggestion
        arch: ["x86_64", "arm64"]
        exclude:
          - version: "18.04"
             architecture: "arm64"
          - architecture: ${{ github.event_name == 'release' && 'arm64' }}
```

I think the 18.04 thing is a genuine bug but OK to defer to another PR since we're not pushing it anyway